clients/drcachesim/simulator/cache_fifo.h

```diff
@@ -54,12 +54,11 @@ class cache_fifo_t : public cache_t {
     {
```

Comment:
> ix: Small Fixes - Consistent Naming, Error Printing, and add_parent.

We do not put "ix: " in the title.  No prefix in the title when there's no issue number.